internal/cli/users.go

```diff
@@ -643,17 +651,15 @@ func formatUserDetailsPath(id string) string {
 	return fmt.Sprintf("users/%s", id)
 }
 
-func (c *cli) connectionPickerOptions() ([]string, error) {
-	var res []string
-
-	list, err := c.api.Connection.List()
+func (c *cli) dbConnectionPickerOptions() ([]string, error) {
+	list, err := c.api.Connection.List(management.Parameter("strategy", management.ConnectionStrategyAuth0))
```

Comment:
We're improving the logic here by directly only retrieving database connections from the API instead of retrieving everything and then filtering them out.